Language reworked (#71)
YunoHost/project-organization: yunohost_project_organization.md

# YunoHost project organisation

## Purpose

> Feel legitimized in contributing to the YunoHost project and part-take in the collective feedback.

Reviewer: ```suggestion
This document aims to make contributors feel legitimized in contributing to the YunoHost project and take part in the collective feedback.
```

> The project is community-based to avoid hasty decisions made by a restricted group of contributors generating frustrations at a later stage.

Reviewer: ```suggestion
Considering that the project is community-based, hasty decisions should be avoided so that everybody can participate and stay aware of what is happening.
```

Decisions are taken collectively, when sufficiently thought out.

> An advisory council orients the evolution of the YunoHost project, and special interest groups allow more efficient contribution in each specific area.

Reviewer: ```suggestion
An advisory council orients the evolution of the YunoHost project, while working groups work on specific topics.
```

## Definition of YunoHost

### Purpose

> Installing and administrating a server should be accessible to the largest number of people, without prejudice to the quality and reliability of the software. 

Reviewer: Hmokay but the initial sentence was clearly stating "The goal of Yunohost is ..." whereas the new sentence sounds more like a wish..?

Author: That part is redundant. In that vein, and as long as the aim is not obviously met, the rest can serve as a reminder of what it should be. I don't understand "without prejudice to the quality and reliability of the software." btw. To me it means the installation and admin part of anything on a server should be accessible, even if the software is bad and unreliable. Meaning YunoHost somehow facilitates bad external software, reading into that on a half-empty glass.

### Values

#### Libre and community-based software

> YunoHost is a software under copyleft libre licence, fully community-based and based on applications which are themselves community-based, and often gratis (Roundcube, Baïkal, etc.)

Reviewer: ```suggestion
YunoHost is a software under copyleft libre licence, fully community-based and integrates with a catalog of applications that are themselves libre (Roundcube, Baïkal, etc.)
```

Author: I shortened this

#### That everyone can appropriate

> Historically, the project has been very close to initiatives aiming to creating a neutral and decentralised Internet. This proximity especially with the [FFDN](https://www.ffdn.org/) materialized when some contributors created the InternetCu.be, whose mission is to facilitate self-hosting by providing a complete solution including a service (via a VPN) and a device. This militant aspect does not inhibit commercial software initiatives, hereby companies could propose support or hosting.

Reviewer: ```suggestion
Historically, the project has been very close to initiatives building and defending a neutral and decentralised Internet. This proximity especially with the [FFDN](https://www.ffdn.org/) was embodied by the creation of the Internet Cube project, whose mission is to facilitate self-hosting by providing a plug-and-play solution including the hardware and a VPN. This militant aspect does not inhibit commercial software initiatives, and companies can propose support for, or hosting based on Yunohost.
```

## YunoHost organisation

### A theme-based, open structure

> The objective of the organisation is to allow the greates number of people to contribute to software improvement, whether from a technical point of view (development, application packaging) or otherwise (communication, end-user assistance, documentation, etc.). Inspired by the projects reviewed at a recent event (Kodi, Debian, Django, Fedora, Wikipedia, etc.) and by ideas stemming from YunoHost contributors (Jérôme, Bram, opi, scith, ju), a decision was made to organise work by special interest groups, federated thanks to a council of key contributors.

Reviewer: ```suggestion
The objective of the organisation is to allow the greatest number of people to contribute to the project, whether from a technical point of view (development, application packaging) or otherwise (communication, end-user assistance, documentation, etc.). Inspired by the projects reviewed before the creation of this document (Kodi, Debian, Django, Fedora, Wikipedia, etc.) and by ideas stemming from YunoHost contributors (Jérôme, Bram, opi, scith, ju), a decision was made to organise work by work groups, federated thanks to a council of key contributors.
```

Author: Filename is organization, content is all organise/organization

Organisational YunoHost project scheme

<img src="https://raw.githubusercontent.com/YunoHost/yunohost-project-organization/master/organization_schema.png" height="600px" />

#### Definition and structure of groups

> Groups are structured as a result of the fact that YunoHost counts many sub-projects (a total of 13), though not always in the knowledge of who is in charge or competent. It has therefore been decided to simplify the organisation into sub-projects according to theme-based groups:

Reviewer: ```suggestion
The purpose of the different working groups is to create teams working on the relatively independents sub-topics of the Yunohost project, and in which contributors can freely organize themselves. The working groups are as follow:
```

Author: Shortened

## Composition of groups

- Council: Bram, ljf, Maniack, Moul, Aleks, frju365, Josue, JimboJoe, Kayou, yalh76
- Core: Aleks, Bram, ljf, JimboJoe, Josue, decentral1se
- Apps: Maniack, frju365, JimboJoe, Josue, ljf, Kayou, yalh76
     - Official apps : Maniack, frju365, JimboJoe, Josue, Kayou, yalh76
- Infra: Bram, ljf, Aleks, Maniack
- Support & Doc: Aleks, ljf, frju365, ppr
- Distribution: Aleks
- Communication: Aleks, frju365, Bram

> - Translation: Quent-i, kingu

Reviewer 2: :wave:

> - Translation: Quent-i, kingu

Author: ```suggestion
- Translation: Quent-i
```
